test/system/260-sdnotify.bats

```diff
@@ -203,7 +203,9 @@ spec:
   restartPolicy: "Never"
   containers:
   - command:
-    - true
+    - /bin/sh
+    - -c
+    - 'while :;do if test -e /tearsinrain; then exit 0;fi;sleep 1;done'
```

Comment:
small nit: could you please add spaces after the semicolons to make it more readable.

Reply:
Done.